roles/internal/webserver-app/tasks/drupal.yml

```diff
@@ -21,11 +21,53 @@
       $content_directories['sync'] = $app_root.'/../content/sync';
     path: "{{ drupal_trusted_hosts_file }}"
     marker: // {mark} ANSIBLE MANAGED BLOCK
+  when: webserver_app_do_original_trusted_host_and_more
+
+- name: Trusted Host Settings
+  blockinfile:
+    state: present
+    block: |
+      $settings['trusted_host_patterns'] = array(
+      {% for host in drupal_trusted_hosts %}
+        '{{ host }}',
+      {% endfor %}
+      );
+    path: "{{ drupal_trusted_hosts_file }}"
+    marker: // {mark} ANSIBLE MANAGED BLOCK - trusted hosts
+  when: webserver_app_do_trusted_host
+
+- name: Flysystem "fedora" scheme configuration
+  blockinfile:
+    state: present
+    block: |
+      $settings['flysystem'] = [
+        'fedora' => [
+          'driver' => 'fedora',
+          'config' => [
+            'root' => '{{ fedora_base_url }}',
+          ],
+        ],
+      ];
+    path: "{{ drupal_site_default_settings }}"
+    marker: // {mark} ANSIBLE MANAGED BLOCK - fedora flysystem scheme
+  when: webserver_app_do_fedora_scheme_config
+
+- name: Content Sync Directory configuration
+  blockinfile:
+    state: present
+    block: |
+      global $content_directories;
+      $content_directories['sync'] = $app_root.'/../content/sync';
+    path: "{{ drupal_site_default_settings }}"
+    marker: // {mark} ANSIBLE MANAGED BLOCK - content sync dir
+  when: webserver_app_do_content_sync_config
```

Comment:
Not sure what the best approach was here, to allow for more granular control over what we being done in what was originally called the "Trusted Host Settings" bit, which was doing more than just setting the "trusted hosts" settings... elected to keep the original, but additionally have all the bits it contained be available in separate chunks, as we don't really want the `content/sync` thing being done in the `starter` "profile".